### DISCUSSION TITLE
Retiring as maintainer

### DISCUSSION THREAD

**Discussion by arjun024** [paketo-buildpacks/feedback#138] [Python Team]
Hi @paketo-buildpacks/python-maintainers 
I'd like to announce I will be retiring from my role as maintainer of the Python subteam effective 2021/Dec/10 (a week from today). I'd like to continue in the subteam as a contributor.

**Comment by arjun024** [comment]
Effective now. cc @paketo-buildpacks/steering-committee

**Comment by dmikusa** [comment]
Thanks for your time as a maintainer. I've moved you from the maintainer to the contributor team.
